File: economy_api/models/BalanceOperation.py
class BalanceOperation:
    def __init__(self, id: str, receiver_id: str, sender_id: str, amount: int, description: str, createdAt: str, updatedAt: str):
        self.id = id
        self.receiver_id = receiver_id
        self.sender_id = sender_id
        self.amount = amount
        self.description = description
        self.createdAt = createdAt
        self.updatedAt = updatedAt

    def __repr__(self):
        return (f"BalanceOperation(id={self.id}, receiver_id={self.receiver_id}, "
                f"sender_id={self.sender_id}, amount={self.amount}, "
                f"description={self.description}, createdAt={self.createdAt}, "
                f"updatedAt={self.updatedAt})")

File: db_migration_service/src/entity/BalanceOperation.ts
import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, UpdateDateColumn } from "typeorm";

@Entity({ name: "balance_operation" })
export class BalanceOperation {
    @PrimaryGeneratedColumn("uuid")
    id!: string;

    @Column({ type: "uuid" })
    receiverId!: string;

    @Column({ type: "uuid" })
    senderId!: string;

    @Column({ type: "integer" })
    amount!: number;

    @Column({ type: "text" })
    description!: string;

    @CreateDateColumn()
    createdAt!: Date;

    @UpdateDateColumn()
    updatedAt!: Date;
}
